src/transaction/lock_manager.c

```diff
@@ -1113,6 +1115,7 @@ lock_initialize_object_hash_table (void)
 
   const int block_count = 2;
   const int block_size = (int) MAX ((lk_Gl.max_obj_locks * LK_RES_RATIO) / block_count, 1);
+  lk_Obj_lock_res_desc.max_alloc_cnt = prm_get_integer_value (PRM_ID_LK_ESCALATION_AT);
```

Comment:
I agree to this change.
But I think it is better to have max_alloc_cnt in LF_FREELIST.
```
/* Add argument: max_alloc_cnt */
lockfree_hashmap<Key, T>::init ()

  /* Add argument: max_alloc_cnt */
  -> init_as_old ()

    /* Add argument: max_alloc_cnt */
    -> m_old_hash.init ()

      /* Add argument: max_alloc_cnt */
      -> lf_freelist_init ()

        freelist->max_alloc_cnt = max_alloc_cnt;
```

Reply:
Why? The max_alloc already exists in the descriptor. If it's in free_list, it's a redundant.

Reply:
My opinion is that it shouldn't be in the descriptor, it should be in the freelist.

Reply:
I don't think so.